app/scripts/templates/settings/clients.mustache

```diff
@@ -75,7 +75,7 @@
       </ul>
 
       <div class="button-row">
-        <button type="submit" class="settings-button primary clients-refresh">{{#t}}Refresh{{/t}}</button>
+        <button type="submit" class="settings-button primary clients-refresh" data-min-progress-indicator-ms="1800">{{#t}}Refresh{{/t}}</button>
```

Comment:
I changed it to 1600 since that's what @ryanfeeley specified in that original bug. Thanks for catching this!